dagger/publish.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"context"
+	"dagger/replicated-sdk/internal/dagger"
+	"fmt"
+)
+
+// Publish publishes the Replicated SDK images and chart
+// to staging and production registries
+//
+//	this is set up to default publish to ttl.sh and that's it
+func (m *ReplicatedSdk) Publish(
+	ctx context.Context,
+
+	// +defaultPath="/"
+	source *dagger.Directory,
+
+	opServiceAccount *dagger.Secret,
+
+	// +optional
+	opServiceAccountProduction *dagger.Secret,
+
+	version string,
+
+	// +default=false
+	staging bool,
+
+	// +default=false
+	production bool,
+
+	// +default=true
+	dev bool,
+
+	// +default=false
+	slsa bool,
+
+	// +default="ttl.sh"
+	chartRegistry string,
+
+	// +optional
+	githubToken *dagger.Secret,
+) error {
+	if production {
+		if err := generateReleaseNotesPR(ctx, source, opServiceAccount); err != nil {
```

Comment:
This is a noop right now , and we're still creating the release notes pr via the gh action - is this one of the things you wanna follow up on and switch to dagger ? If not - maybe we should nuke it?